IdentityServer/v6/Configuration/Configuration/Program.cs

```diff
@@ -0,0 +1,47 @@
+using Configuration;
+using Duende.IdentityServer.Configuration;
+using Duende.IdentityServer.Configuration.EntityFramework;
+using Duende.IdentityServer.EntityFramework.DbContexts;
+using Duende.IdentityServer.EntityFramework.Storage;
+using Duende.IdentityServer.Services;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddIdentityServerConfiguration(opt => {})
+    .AddClientConfigurationStore();
+
+var connectionString = builder.Configuration.GetConnectionString("DefaultConnection");
+builder.Services.AddConfigurationDbContext<ConfigurationDbContext>(options =>
+{
+    options.ConfigureDbContext = builder => builder.UseSqlite(connectionString);
+});
+
+builder.Services.AddAuthentication("token")
+    .AddJwtBearer("token", options =>
+    {
+        options.Authority = "https://localhost:5001";
+        options.MapInboundClaims = false;
+        
+        options.TokenValidationParameters.ValidateAudience = false;
+        options.TokenValidationParameters.ValidTypes = new[] { "at+jwt" };
+    });
+
+// TODO - simplify adding these services
+builder.Services.TryAddTransient<ICancellationTokenProvider, DefaultHttpContextCancellationTokenICancellationTokenProvider>();
+builder.Services.TryAddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+
```

Comment:
Need to make this happen automatically when you call AddClientConfigurationStore...